### PR TITLE
Add examples for asInputStream and asOutputStream

### PR DESCRIPTION
**Pull request by nitikagarw**
References #25468

### CODE REVIEW

**Comment by nitikagarw** [comment]
@johanandren @patriknw Please review

File: akka-docs/src/main/paradox/stream/operators/StreamConverters/asInputStream.md [hunk 1]
```diff
@@ -25,3 +25,10 @@ The `InputStream` will be ended when the stream flowing into this `Sink` complet
 **backpressures** when no read is pending on the `InputStream`
 @@@
 
+## Example
+
```
Comment (johanandren) [inline akka-docs/src/main/paradox/stream/operators/StreamConverters/asInputStream.md:29]: Please include a short textual explanation of what the example does here

File: akka-docs/src/main/paradox/stream/operators/StreamConverters/asOutputStream.md [hunk 2]
```diff
@@ -24,3 +24,10 @@ closing the `OutputStream` will complete the `Source`.
 **completes** when the `OutputStream` is closed
 @@@
 
+## Example
+
```
Comment (johanandren) [inline akka-docs/src/main/paradox/stream/operators/StreamConverters/asOutputStream.md:28]: Same here, short explanation

File: akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java [hunk 3]
```diff
@@ -80,4 +86,41 @@ public void demonstrateFromJavaIOStreams()
...
         new String(((ByteArrayOutputStream) outputStream).toByteArray(), charset));
   }
+
+  @Test
+  public void demonstrateAsJavaInputStream() throws Exception {
+
+    // #asJavaInputStream
+    final Sink<ByteString, InputStream> sink = StreamConverters.asInputStream();
+    final List<ByteString> list =
+        Collections.singletonList(ByteString.fromString("Some random input"));
```
Comment (johanandren) [inline akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java:96]: No need for this intermediate `List`,  use `Source.single` just like in the Scala sample.

**Review by johanandren — COMMENTED** [review]
Thanks, I would have wished for more realistic/less contrived samples, I don't have any concrete ideas right now though and this is better than no examples at all.

Added some feedback for small improvements.

File: akka-docs/src/main/paradox/stream/operators/StreamConverters/asInputStream.md (hunk 1 above)
Comment (nitikagarw) [reply akka-docs/src/main/paradox/stream/operators/StreamConverters/asInputStream.md:30]: Done

File: akka-docs/src/main/paradox/stream/operators/StreamConverters/asOutputStream.md (hunk 2 above)
Comment (nitikagarw) [reply akka-docs/src/main/paradox/stream/operators/StreamConverters/asOutputStream.md:29]: Done

File: akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java (hunk 3 above)
Comment (nitikagarw) [reply akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java]: Done

**Review by johanandren — APPROVED** [review]
LGTM

**Review by raboof — APPROVED** [review]

**Pull request merged**